packages/commons/src/config/httpServiceConfig.ts

```diff
@@ -13,10 +13,13 @@ export const JWTConfig = z
       .string()
       .transform((s) => s.split(","))
       .pipe(z.array(z.string())),
+
+    JWKS_CACHE_MAX_AGE: z.coerce.number().optional(),
```

Comment:
minor but easier when adding the config in another repo
```suggestion
    JWKS_CACHE_MAX_AGE_MILLIS: z.coerce.number().optional(),
```